docs/cubeviz/plugins.rst

```diff
@@ -287,13 +287,13 @@ spectral region of interest.
 
 .. _spectral-extraction:
 
-Spectral Extraction
-===================
+3D Spectral Extraction
+========================
```

Comment:
```suggestion
======================
```
I think this is too many `=`, should be the same as the number of characters. I don't know if it actually affects rendering or not. I'm guessing this is from when it was "Cube" instead of "3D".